bindings/py/cpp_src/bindings/encoders/encoders_module.cpp

```diff
@@ -30,12 +30,32 @@ namespace py = pybind11;
 namespace nupic_ext
 {
     void init_ScalarEncoder(py::module&);
+    void init_RDSE(py::module&);
 }
 
 using namespace nupic_ext;
 
 PYBIND11_MODULE(encoders, m) {
-    m.doc() = "nupic.bindings.encoders"; // optional module docstring
+    m.doc() =
+R"(nupic.bindings.encoders
+
+An encoder converts a value to a sparse distributed representation.
+
+There are several critical properties which all encoders must have:
+
+    1) Semantic similarity:  Similar inputs should have high overlap.  Overlap
+    decreases smoothly as inputs become less similar.  Dissimilar inputs have
+    very low overlap so that the output representations are not easily confused.
+
+    2) Stability:  The representation for an input does not change during the
+    lifetime of the encoder.
+
+    3) Sparsity: The output SDR should have a similar sparsity for all inputs and
```

Comment:
I think of encoders as producing SDRs because they have many of the right properties. But yes as you point out they dont always have the distributed/decorrelated property.